### DISCUSSION TITLE
Can I add a control rig to an already imported meta rig?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [soupday/cc3_blender_tools#246] [General]
Hi, if I already have a skeleton in Blender thats compatible with reallusions bone structure, do I need to like export to fbx and then import again to generate a control rig from it (or is there a button somewhere for adding control rig to existing meta rig)?

**Comment by soupday** [comment]
The rigify process in this add-on is for the CC3+ standard skeleton. So it would have to come from CC3/4 with the standard bone names to be compatible with that. i.e. the ActorBuild or G3 skeleton if you're converting a custom humanoid.
